lib/classes/popover.js

```diff
@@ -70,20 +70,32 @@ class PopOver extends ToolTip {
     getTitle() {
         let title = this.$config.title || '';
         if (typeof title === 'function') {
-            title = title.call(this.$element).toString();
+            title = title.call(this.$element).toString().trim();
+        }
+        if (typeof title === 'object' && title.nodeType && !title.innerHTML.trim()) {
+            // We have a dom node, but without inner content, so just return an empty string
+            title = '';
         }
         if (!title) {
+            // Try and grab elements title attribute
             title = this.$element.getAttribute('title') || this.$element.getAttribute('data-original-title') || '';
+            title = title.trim();
         }
-        return title.trim();
+        return title;
     };
 
     // New methods
 
     getContent() {
-        return typeof this.$config.content === 'function' ?
-              this.$config.content.call(this.$element) :
-              this.$config.content;
+        let content = this.$config.content || '';
+        if (typeof content === 'function') {
+            content = content.call(this.$element).toString.trim();
```

Comment:
This isnt work with current example. Function returns object, and then it stringified to [object Object]

Reply:
This when setting content to a function, or passing a dom object?

Reply:
when function returns object